src/main/java/net/sf/jabref/wizard/auximport/AuxSubGenerator.java

```diff
@@ -224,8 +222,8 @@ private boolean parseAuxFile(String filename) {
                             }
                         }
                     }
-                } // line != null
-                else {
+                } else {
+                    // line != null
                     weiter = false;
```

Comment:
"weiter" should be renamed to "continue" for people who don't speak German :wink: